packages/assertions/src/subtle-crypto.ts

```diff
@@ -1,10 +1,17 @@
+import {
+    SOLANA_ERROR__SUBTLE_CRYPTO_DIGEST_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_ED25519_ALGORITHM_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_EXPORT_FUNCTION_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_GENERATE_FUNCTION_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_SIGN_FUNCTION_MISSING,
+    SOLANA_ERROR__SUBTLE_CRYPTO_VERIFY_FUNCTION_MISSING,
+    SolanaError,
+} from '@solana/errors';
+
 function assertIsSecureContext() {
     if (__BROWSER__ && !globalThis.isSecureContext) {
-        // TODO: Coded error.
-        throw new Error(
-            'Cryptographic operations are only allowed in secure browser contexts. Read more ' +
-                'here: https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts',
-        );
+        throw new SolanaError(SOLANA_ERROR__SUBTLE_CRYPTO_MISSING);
```

Comment:
I mean, as a corollary yes, but the actual error is `SOLANA_ERROR__CRYPTO_DISALLOWED_IN_INSECURE_CONTEXT` or something.

Reply:
…like, `SubtleCrypto` can be missing for other reasons, like the environment is otherwise ‘secure’ but just doesn't support `SubtleCrypto`.

Reply:
I went for `SOLANA_ERROR__SUBTLE_CRYPTO_DISALLOWED_IN_INSECURE_CONTEXT` in the feedback PR.

I also changes the `_MISSING` suffixes with `_UNIMPLEMENTED` as you suggested and added a description for it in the comments.